src/components/attendee/VideoHuddle/useVideoHuddle.tsx

```diff
@@ -1,24 +1,103 @@
-import { useContext } from "react";
+import React, { useContext, useMemo, useState } from "react";
+import { IconDefinition } from "@fortawesome/free-solid-svg-icons";
 
-import { VideoHuddleContext } from "./VideoHuddle";
+import { useVideoComms } from "../VideoComms/hooks";
+import { VideoTrack } from "../VideoComms/types";
+
+interface HuddleContextType {
+  inHuddle: boolean;
+  setInHuddle: (cb: (prevInHuddle: boolean) => void) => void;
```

Comment:
Should we just not make callbacks accept objects? Something like `ctx: SetInHuddleContext` that maybe has only one field now, but can be expanded later on w/o rewriting other listeners. This way we can pass more info as new use cases arise.